internal/graphql/schema.resolvers.go

```diff
@@ -37,6 +37,10 @@ import (
 	"go.uber.org/zap"
 )
 
+const CONFIGURATION_METRICS_UPDATE_INTERVAL = 10
+const OVERVIEW_METRICS_UPDATE_INTERVAL = 10
+const AGENT_METRICS_UPDATE_INTERVAL = 10
```

Comment:
```suggestion
const CONFIGURATION_METRICS_UPDATE_INTERVAL = 10 * time.Second
const OVERVIEW_METRICS_UPDATE_INTERVAL = 10 * time.Second
const AGENT_METRICS_UPDATE_INTERVAL = 10 * time.Second
```

Make these the full time.Duration and update the code below that uses them.